test/clj_sqs_extended/core_test.clj

```diff
@@ -16,49 +16,59 @@
 
 (deftest can-receive-message
   (testing "Sending/Receiving basic maps"
-    (let [client (fixtures/localstack-sqs)
-          url (fixtures/test-queue-url)
-          test-message (first test-messages)]
-      (doseq [format [:transit :json]]
-        (sqs-ext/send-message client url test-message {:format format})
-        (let [message (sqs-ext/receive-message client url {:format format})]
-          (is (= test-message (:body message))))))))
+    (fixtures/with-standard-queue
+      (let [test-message (first test-messages)]
+        (doseq [format [:transit :json]]
+          (sqs-ext/send-message @fixtures/test-ext-sqs-client
+                                @fixtures/test-standard-queue-url
```

Comment:
using defonce like what you do here could be a way. I would change things around a bit to make it more intuitive. Perhaps declare these 2 vars in this namespace with `defonce` like you did, but pass them into proper fixture `fixtures/with-test-ext-sqs-client` as arguments to be setup and destroyed as part of the fixture. Then the rest of your test code can eval them like,

```
(sqs-ext/send-message @test-ext-sqs-client @test-standard-queue-url)
```

Otherwise, currently `@fixtures/test-ext-sqs-client @fixtures/test-standard-queue-url` pops out of nowhere from perspective of the reader to this test.

Thoughts?

This is a small issue. If it's not a quick fix, I'd just move on for now and come back later.

Reply:
Saved to issues and moving on for now.

Reply:
This is tricky. There's an issue for this to be solved later. For now lets continue with the previous version.